sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -317,6 +317,7 @@ private boolean matchStatus(PollResponse<T> currentPollResponse, OperationStatus
             .onErrorResume(throwable -> {
                 // We should never get here and since we want to continue polling
                 //Log the error
+                logger.warning("Failed to apply delay and call poll operation. Details: {}" + throwable.getMessage());
```

Comment:
Instead of logging the message manually, pass the exception as a parameter. Our logger should be exposing these details.

Reply:
Done.